wp-content/themes/twentyseventeen/Custom_Template.php

```diff
@@ -0,0 +1,3 @@
+<?php
+//Template Name : Custom_Template
```

Comment:
This file is need when we create custom template page so we need to write in comment for activate this page